algorithm_config.yaml

```diff
@@ -1,6 +1,6 @@
 description: Subset GEDI L1B, L2A, L2B, or L4A granules within an area of interest (AOI)
 algo_name: gedi-subset
-version: develop
+version: 0.4.0
```

Comment:
Leave this unchanged. We don't bump the version on branches merged into develop. We bump the version in the develop branch, and only just prior to merging develop into main. And then after merging develop into main, we make another version change here back to develop on the develop branch so that all branches off of develop always have develop as the version and we don't have any merge conflicts on this value when merging multiple branches back to develop.